Directory.Build.targets

```diff
@@ -15,6 +15,7 @@
     <StrongNameKeyId Condition="'$(IsTestProject)' == 'true'">$(TestStrongNameKeyId)</StrongNameKeyId>
   </PropertyGroup>
 
+  <Import Project="$(RepositoryEngineeringDir)Resources.targets" />
```

Comment:
@natemcmaster it is unfortunate that the Arcade change now relies on the importing order. Is there a way to fix this?

Reply:
In this particular case, it's tricky. To make Visual Studio editing of .resx files work well, we need to set metadata on an item during static evaluation. There is no perfect solution for import ordering when it comes to static evaluation.